addons/crm/data/crm_lead_merge_post_message_template.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="crm_lead_merge_summary" name="crm_lead_merge_summary">
+    <div class="o_crm_merge_post_message">
+        <t t-foreach="opportunities" t-as="lead">
+            <div>
+                <span>Merged the Lead/Opportunity</span>
+                <span class="font-weight-bold" t-esc="lead.name"/>
+                <span>into this one.</span>
+            </div>
+            <blockquote class="o_crm_merge_content" data-o-mail-quote-node="1" data-o-mail-quote="1">
```

Comment:
Isn't `data-o-mail-quote` enough ? (`data-o-mail-quote-node` is useless I think)

Reply:
Indeed, it works fine with only "data-o-mail-quote".